src/components/learner-credit-management/BudgetCard.jsx

```diff
@@ -1,80 +1,102 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { useOfferSummary } from './data';
+import { useSubsidySummaryAnalyticsApi } from './data';
 import SubBudgetCard from './SubBudgetCard';
 import { BUDGET_TYPES } from '../EnterpriseApp/data/constants';
 
 /**
- * Renders one or more budget cards for the given offer (enterprise or Subsidy from enterprise-subsidy). If the offer is
- * an enterprise offer, it will render a single card. If the offer is a Subsidy, it will render one card for
- * each associated budget.
+ * Renders one or more budget cards for the given budget. If the budget is
+ * an enterprise offer, it will render a single card. If the budget is a Subsidy,
+ * it will render one card for each associated budget. If the budget is a Policy,
+ * it will also render a single card.
+ *
+ * @param {Object} budget Represents either:
+ *  - Enterprise Offer
+ *  - Subsidy (enterprise-subsidy)
+ *  - Policy (enterprise-access)
  *
- * @param {*} offer Represents either an enterprise offer or a Subsidy (enterprise-subsidy).
  * @returns Budget card component(s).
  */
 const BudgetCard = ({
-  offer,
+  budget,
   enterpriseUUID,
   enterpriseSlug,
-  offerType,
-  displayName,
 }) => {
-  const { start, end } = offer;
-
   const {
-    isLoading: isLoadingOfferSummary,
-    offerSummary,
-  } = useOfferSummary(enterpriseUUID, offer);
+    isLoading: isLoadingSubsidySummaryAnalyticsApi,
+    subsidySummary: subsidySummaryAnalyticsApi,
+  } = useSubsidySummaryAnalyticsApi(enterpriseUUID, budget);
+
+  // Subsidy Access Policies will always have a single budget, so we can render a single card
+  // without relying on `useSubsidySummaryAnalyticsApi`.
+  if (budget.source === BUDGET_TYPES.policy) {
+    return (
+      <SubBudgetCard
+        id={budget.id}
+        start={budget.start}
+        end={budget.end}
+        available={budget.aggregates.available}
+        spent={budget.aggregates.spent}
+        pending={budget.aggregates.pending}
+        displayName={budget.name}
+        enterpriseSlug={enterpriseSlug}
+      />
+    );
+  }
 
   // Enterprise Offers will always have a single budget, so we can render a single card.
```

Comment:
[nit] similar to the one above, I would also append the api source after Enterprise Offers